providers/common/compat/src/airflow/providers/common/compat/sdk.py

```diff
@@ -90,6 +90,7 @@
         TaskDeferred as TaskDeferred,
         XComNotFound as XComNotFound,
     )
+    from airflow.sdk.log import redact as redact
     from airflow.sdk.observability.stats import Stats  # noqa: F401
```

Comment:
Unrelate to this PR, could you fix the pattern on this line too to get rid of noqa?